app/modules/individuals/resources.py

```diff
@@ -101,6 +101,14 @@ def post(self, args):
         ):
             cleanup.rollback_and_abort(message='No Encounters in POST')
 
+        from app.modules.encounters.models import Encounter
+
+        for enc_json in request_in['encounters']:
+            encounter = Encounter.query.get(enc_json['id'])
```

Comment:
if 'id' is not present in enc_json, this will fail and I think the cleanup will not happen